ext/RMagick/rmdraw.c

```diff
@@ -1067,239 +1067,28 @@ Draw_composite(int argc, VALUE *argv, VALUE self)
-                break;
-            case LinearDodgeCompositeOp:
-                op = "LinearDodge";
-                break;
-            case LinearLightCompositeOp:
-                op = "LinearLight";
-                break;
-            case LuminizeCompositeOp:
-                op = "Luminize";
-                break;
-            case MathematicsCompositeOp:
-                op = "Mathematics";
-                break;
-            case MinusCompositeOp:
-                op = "Minus";
-                break;
-            case MinusSrcCompositeOp:
-                op = "MinusSrc";
-                break;
-            case ModulateCompositeOp:
-                op = "Modulate";
-                break;
-            case MultiplyCompositeOp:
-                op = "Multiply";
-                break;
-            case NoCompositeOp:
-                op = "No";
-                break;
-            case OutCompositeOp:
-                op = "Out";
-                break;
-            case OverCompositeOp:
-                op = "Over";
-                break;
-            case OverlayCompositeOp:
-                op = "Overlay";
-                break;
-            case PegtopLightCompositeOp:
-                op = "PegtopLight";
-                break;
-            case PinLightCompositeOp:
-                op = "PinLight";
-                break;
-            case PlusCompositeOp:
-                op = "Plus";
-                break;
-            case ReplaceCompositeOp:
-                op = "Replace";
-                break;
-            case SaturateCompositeOp:
-                op = "Saturate";
-                break;
-            case ScreenCompositeOp:
-                op = "Screen";
-                break;
-            case SoftLightCompositeOp:
-                op = "SoftLight";
-                break;
-            case SrcCompositeOp:
-                op = "Src";
-                break;
-            case SrcAtopCompositeOp:
-                op = "SrcAtop";
-                break;
-            case SrcInCompositeOp:
-                op = "SrcIn";
-                break;
-            case SrcOutCompositeOp:
-                op = "SrcOut";
-                break;
-            case SrcOverCompositeOp:
-                op = "SrcOver";
-                break;
-            case SubtractCompositeOp:
-                op = "Subtract";
-                break;
-            case ThresholdCompositeOp:
-                op = "Threshold";
-                break;
-            case UndefinedCompositeOp:
-                op = "Undefined";
-                break;
-            case VividLightCompositeOp:
-                op = "VividLight";
-                break;
-            case XorCompositeOp:
-                op = "Xor";
-                break;
-            default:
-                rb_raise(rb_eArgError, "unknown composite operator (%d)", cop);
-                break;
-        }
+    op=CommandOptionToMnemonic(MagickComposeOptions, cop);
```

Comment:
I prefer the spaces before/after `=`.

```suggestion
    op = CommandOptionToMnemonic(MagickComposeOptions, cop);
```

Reply:
Copy pasted this from ImageMagick where we don't do this. Thanks for the easy change.